src/components/Chat.tsx

```diff
@@ -53,11 +56,12 @@ const getToolStatus = (
 export function Chat() {
   const messagesEndRef = useRef<HTMLDivElement>(null)
   const [hasStartedChat, setHasStartedChat] = useState(false)
+  const [focusTimestamp, setFocusTimestamp] = useState(Date.now())
   const [servers, setServers] = useState<Servers>({})
   const [selectedServers, setSelectedServers] = useState<string[]>([])
   const [streamBuffer, setStreamBuffer] = useState<StreamEvent[]>([])
   const [streaming, setStreaming] = useState(false)
-  const { selectedModel } = useModel()
+  const { selectedModel, setSelectedModel } = useModel()
```

Comment:
`setSelectedModel` is here now because the model dropdown is in the chat component instead of the header now.